sbin/prepareWorkspace.sh

```diff
@@ -336,7 +336,21 @@ checkingAndDownloadingAlsa() {
     # Note: the uptime command below is to aid diagnostics for this issue:
     # https://github.com/adoptium/temurin-build/issues/3518#issuecomment-1792606345
     uptime
-    gpg --keyserver keyserver.ubuntu.com --keyserver-options timeout=300 --recv-keys "${ALSA_LIB_GPGKEYID}"
+    # Will retry command below until it passes or we've failed 10 times.
```

Comment:
the comment does not seem to be aligned with the code. Here its written that we try 10 times, but the code below seem to try 3 different keyservers once.

Reply:
We were trying to access one server 10 times, but this comment was not updated when I changed that. Will fix. Thank you. :)